vscode/microsoft-kiota/src/kiotaInterop/types.ts

```diff
@@ -0,0 +1,257 @@
+export enum KiotaGenerationLanguage {
```

Comment:
Isn't http supposed to be part of this?

Reply:
Not yet... the support work on VS Code hasn't been done